### ISSUE TITLE
Disrepancy in architecture in paper vs implemented version

### ISSUE THREAD

**Issue by Froskekongen** [jacobgil/keras-dcgan#5]
To me it seems that the architecture in the referenced paper is quite different from the one that is implemented right now. 

In particular: 
- the use of relu and leakyrely activations.
- batchnormalisation between the layers
- striding instead of pooling

BTW: I really like this initiative (looking to use adversarial nets within keras quite soon.)

**Comment by brettbj** [comment]
The paper also uses Adam instead of SGD
